ui/ducks/send/send.js

```diff
@@ -1090,33 +1219,9 @@ const slice = createSlice({
       .addCase(GAS_FEE_ESTIMATES_UPDATED, (state, action) => {
         // When the gasFeeController updates its gas fee estimates we need to
         // update and validate state based on those new values
-        const { gasFeeEstimates, gasEstimateType } = action.payload;
```

Comment:
Code that serves the purpose of this set of removed lines is added to `updateGasFeeEstimates` (with appropriate modifications for getting the send slice to work with EIP1559 networks)